src/components/rating-dialog.js

```diff
@@ -1,26 +1,27 @@
 import PropTypes from 'prop-types';
-import { Modal, Spin } from 'antd';
-import Markdown from './markdown.js';
 import Logger from '../common/logger.js';
 import StarRating from './star-rating.js';
+import { useUser } from './user-context.js';
 import DeleteButton from './delete-button.js';
 import { useTranslation } from 'react-i18next';
-import { useDateFormat } from './locale-context.js';
+import DocumentRating from './document-rating.js';
+import { Divider, Modal, Progress, Spin } from 'antd';
 import { handleApiError } from '../ui/error-helper.js';
 import { useSessionAwareApiClient } from '../ui/api-helper.js';
-import React, { useCallback, useEffect, useState } from 'react';
 import { documentRatingShape } from '../ui/default-prop-types.js';
+import React, { Fragment, useCallback, useEffect, useState } from 'react';
 import DocumentRatingApiClient from '../api-clients/document-rating-api-client.js';
 
 const logger = new Logger(import.meta.url);
 
-function RatingDialog({ documentRating, isOpen, onCancel, onOk }) {
-  const { formatDate } = useDateFormat();
+const STAR_STROKE_COLOR = 'rgb(250, 219, 20)';
```

Comment:
Just a thought, no changes suggested: I'm split between doing this "the ant way" which is not the educandu way, or working against ant and overwriting the `background-color` of the `ant-progress-bg` CSS class 😅

Reply:
I had the same thought, but as the star color is hard-coded in Antd (and therefore we do not have it in CSS anywhere), I went the Antd way here too.